d2/src/main/java/com/linkedin/d2/balancer/servers/ZooKeeperAnnouncer.java

```diff
@@ -211,8 +219,102 @@ public void onSuccess(None result)
         }
         runNextMarkUpOrMarkDown();
       }
-    });
+    };
+
+    final Callback<None> doMarkUpCallback = new Callback<None>() {
```

Comment:
Looks like the style in this file is to have the opening curly brace on a separate line.  It would be good to match the existing style & move your curly brace down onto the next line (unless the curly brace has to stay on the same line to compile correctly)

Reply:
Thanks! Done.